CHANGES.md

```diff
@@ -3,7 +3,7 @@ Release Changes List
 ====================
 
 ==============
-version 1.13.0
+version 1.13.1 (PRE-RELEASE)
```

Comment:
@awallace-cray: As I confessed yesterday, I didn't actually review this patch, just the approach (committing to master vs. the branch).  This instance of 1.13.0 shouldn't have changed because CHANGE.md represents a running log of changes with each release, so new entries should be added to the top without changing old entries.  I'm fixing this in my changes-1.13.1 branch.